docs/src/getting-started/custom_dataset_conf.rst

```diff
@@ -78,7 +77,7 @@ A single string in this section automatically expands, using the string as the
 
 .. note::
 
-   ``metatensor-models`` does not convert units during training or evaluation. Units are
+   `metatensor-models` does not convert units during training or evaluation. Units are
```

Comment:
It seems to me that single backticks are for italic and double backticks are for code-like fonts. Are you sure we want italic here?

Reply:
I am using an italic font if the library name does not appear as a code example, like it is written here. We can change this if you like.

Reply:
It looks weird to me when I read the documentation but it's a matter of taste